### DISCUSSION TITLE
How to use in nuxt3

### DISCUSSION THREAD

**Discussion by likymm** [NathanAP/vue-google-maps-community-fork#29] [Q&A] [answered]
Please help, I'm supper new to Vue and Nuxt 3.

I'm currently using [@fawmi/vue-google-maps](https://github.com/fawmi/vue-google-maps) and since lack of support, I want to use this library. But, I'm getting an error when using it on Nuxt 3.

```console
nuxt: ^3.0.0
vue-google-maps-community-fork: ^0.1.6
```
I also got this error on `@fawmi/vue-google-maps` but at least I have these [work around](https://github.com/fawmi/vue-google-maps/issues/164) by pinning this version `"@fawmi/vue-google-maps": "0.9.72",`

I'm not getting error but I only see white screen,

Also, based on [Nuxt 3 Plugins Directory Guide](https://nuxt.com/docs/guide/directory-structure/plugins#plugins-directory), I placed my plugin in `/plugins` not `/plugin` opposed to the example in this docs for Nuxt 3.

Thanks in advance.

**Comment by NathanAP** [comment, marked as answer]
We are following this in #33 so I will lock this one.
